### PR TITLE
Introduce CaptureFinished message

### PR DESCRIPTION
**Pull request by dimitry-**
This is last event sent for the capture. This
commit establishes the message and sends it at the
end of the capture. For now it is always a success.

Bug: http://b/185754690
Test: run capture in the client.

### CODE REVIEW

**Review by florian-kuebler — APPROVED** [review]
LGTM. Thanks.

Just one question: Deserialization will not be affected by those new messages, right? Do we make sure that this remains without problems?

**Comment by dimitry-** [comment]
> Just one question: Deserialization will not be affected by those new messages, right? Do we make sure that this remains without problems?

Actually, maybe it should. Looking.

**Comment by dimitry-** [comment]
> > Just one question: Deserialization will not be affected by those new messages, right? Do we make sure that this remains without problems?
> 
> Actually, maybe it should. Looking.

Ok, it doesn't really have any effect on capture deserialization for current format. Going to leave it as it is.

**Comment by florian-kuebler** [comment]
> > > Just one question: Deserialization will not be affected by those new messages, right? Do we make sure that this remains without problems?
> > 
> > 
> > Actually, maybe it should. Looking.
> 
> Ok, it doesn't really have any effect on capture deserialization for current format. Going to leave it as it is.

Agree. Just wanted to have this checked.

File: src/Service/CaptureServiceImpl.cpp [hunk 1]
```diff
@@ -200,6 +201,13 @@ static void StopInternalProducersAndCaptureStartStopListenersInParallel(
   }
 }
 
+static ClientCaptureEvent CreateCaptureFinishedEvent() {
```
Comment (dpallotti) [inline src/Service/CaptureServiceImpl.cpp:204]: Maybe move after `CreateCaptureStartedEvent` as it seems more logical.

**Review by dpallotti — APPROVED** [review]

File: src/Service/CaptureServiceImpl.cpp (hunk 1 above)
Comment (dimitry-) [reply src/Service/CaptureServiceImpl.cpp:204]: done.

**Pull request merged**